app/lib/database/facade/Database.ts

```
@@ -0,0 +1,126 @@
+/**
+ * Facade Database class.
+ *
+ * Constructed from a DbHandle (from driver/connection.ts).
+ * Exposes: get(table) → Collection, write(fn), batch(...models), unsafeResetDatabase().
+ */
+
+import { eq, getTableColumns } from 'drizzle-orm';
+import type { SQLiteTable } from 'drizzle-orm/sqlite-core';
```

ESLint

[error] '/home/jailuser/git/node_modules/drizzle-orm/sqlite-core/index.js' imported multiple times.

(import/no-duplicates)